modules/navier_stokes/test/tests/finite_volume/wcns/natural_convection/tests

```diff
@@ -9,5 +9,6 @@
     abs_zero = 1e-9
     # skip test if test is being run out-of-tree. Issue Ref: #26129
     installation_type = in_tree
+    rel_err = 1e-4
```

Comment:
``` test:finite_volume/wcns/natural_convection.natural_circulation_pipe:   --------- Time step 13, 4.0950000e+02 ~ 4.0950000e+02, rel diff:  0.00000e+00 ---------
test:finite_volume/wcns/natural_convection.natural_circulation_pipe: Global variables:
test:finite_volume/wcns/natural_convection.natural_circulation_pipe: Element variables:
test:finite_volume/wcns/natural_convection.natural_circulation_pipe:   --------- Time step 14, 8.1910000e+02 ~ 8.1910000e+02, rel diff:  0.00000e+00 ---------
test:finite_volume/wcns/natural_convection.natural_circulation_pipe: Global variables:
test:finite_volume/wcns/natural_convection.natural_circulation_pipe: Element variables:
test:finite_volume/wcns/natural_convection.natural_circulation_pipe:   --------- Time step 15, 1.6383000e+03 ~ 1.6383000e+03, rel diff:  0.00000e+00 ---------
test:finite_volume/wcns/natural_convection.natural_circulation_pipe: Global variables:
test:finite_volume/wcns/natural_convection.natural_circulation_pipe: Element variables:
test:finite_volume/wcns/natural_convection.natural_circulation_pipe:    superficial_vel_y   rel diff:  9.4482463e-01 ~  9.4480632e-01 = 1.93788e-05 (block 0, elmt 7)
test:finite_volume/wcns/natural_convection.natural_circulation_pipe:   --------- Time step 16, 3.2767000e+03 ~ 3.2767000e+03, rel diff:  0.00000e+00 ---------
test:finite_volume/wcns/natural_convection.natural_circulation_pipe: Global variables:
test:finite_volume/wcns/natural_convection.natural_circulation_pipe: Element variables:
test:finite_volume/wcns/natural_convection.natural_circulation_pipe:    superficial_vel_y   rel diff:  9.4483725e-01 ~  9.4480851e-01 = 3.04157e-05 (block 0, elmt 7)
test:finite_volume/wcns/natural_convection.natural_circulation_pipe:   --------- Time step 17, 6.5535000e+03 ~ 6.5535000e+03, rel diff:  0.00000e+00 ---------
test:finite_volume/wcns/natural_convection.natural_circulation_pipe: Global variables:
test:finite_volume/wcns/natural_convection.natural_circulation_pipe: Element variables:
test:finite_volume/wcns/natural_convection.natural_circulation_pipe:    superficial_vel_y   rel diff:  9.4483725e-01 ~  9.4480851e-01 = 3.04157e-05 (block 0, elmt 7)
test:finite_volume/wcns/natural_convection.natural_circulation_pipe:   --------- Time step 18, 1.0000000e+04 ~ 1.0000000e+04, rel diff:  0.00000e+00 ---------
test:finite_volume/wcns/natural_convection.natural_circulation_pipe: Global variables:
test:finite_volume/wcns/natural_convection.natural_circulation_pipe: Element variables:
test:finite_volume/wcns/natural_convection.natural_circulation_pipe:    superficial_vel_y   rel diff:  9.4483725e-01 ~  9.4480851e-01 = 3.04157e-05 (block 0, elmt 7)
test:finite_volume/wcns/natural_convection.natural_circulation_pipe: 
test:finite_volume/wcns/natural_convection.natural_circulation_pipe: exodiff: Files are different```

Reply:
This can actually come from the change in the piece-wise linear function above, which might or might not have been intended by the previous author. Maybe the benchmark is for fluids with no viscosity. I would change the input file back and remove the relative error override.

Reply:
Since i need to divide the Darcy friction factor with mu to make it consistent with the previous kernel and don't change the gold file, i can't have mu = 0.0. That changes the results a little bit so i had to increase the relative error to 1e-4, which i think is sufficient.